build-constraints.yaml

```
@@ -4204,8 +4204,11 @@ packages:
 
     "Sergey Vinokurov <serg.foo@gmail.com> @sergv":
         - bencoding
-        - emacs-module < 0 # 0.1.1 compile fail ghc 8.10.1 #5436/closed
+        - emacs-module
         - tasty-ant-xml
+        - prettyprinter-combinators
+        - htoml-parse
+        - htoml-megaparsec-parse
```

I think this should be

```suggestion
        - htoml-megaparsec
```

(looking at the CI failure, it's either this typo or a new package that's not yet on hackage)

---

Thanks!